src/main/paradox/docs/api/assets/realms/realm-added-ref.json

```diff
@@ -0,0 +1,17 @@
+{
+  "@context": "https://bluebrain.github.io/nexus/contexts/realms.json",
+  "@id": "http://dev.nexus.ocp.bbp.epfl.ch/auth/realms/realm1",
+  "@type": "nxv:realm",
+  "name": "realm1",
```

Comment:
is this in line with admin? I remember the use of `label` instead of name, but I may be mistaken